synapse/lib/datapath.py

```diff
@@ -92,7 +92,7 @@ def _elem_iter(self):
         # of their names...
         if type( self._d_item ) == dict:
             for name,item in self._d_item.items():
-                yield initelem(item,name=name,parent=self)
+                yield initelem({0: name, 1: item},name=name,parent=self)
```

Comment:
close.  try out:
```
yield initelem((name,item), ...
```
No need for it to be an actual dictionary when 0 and 1 are indexes into a tuple anyway